ckanext/restricted/tests/test_access_request_email_templates.py

```diff
@@ -0,0 +1,64 @@
+# encoding: utf-8
+import ckan.tests.factories as factories
+from ckan.lib.helpers import url_for
+import pytest
+import mock
+
+
+@pytest.mark.usefixtures(u'clean_db')
+@pytest.mark.usefixtures(u'clean_index')
+@pytest.mark.ckan_config(u'ckan.plugins', u'restricted image_view recline_view')
+@pytest.mark.usefixtures(u'with_plugins')
+@pytest.mark.usefixtures(u'with_request_context')
+class TestAccessRequestEmailTemplate(object):
+
+    @mock.patch('ckan.lib.mailer.mail_recipient')
+    def test_restricted_access_request_template_encoding(self, mocked_mail_recipient, app):
```

Comment:
This test doesn't test the actual bug reported by user. It should check if generated resource link is a valid link (returning 200 and displaying resource edit form). It shouldn't check explicitly for `assert '&amp;' not in email_body`
What if the URL suddenly include `localhost` instead of `CKAN_URL` or include &nbsp; etc. etc.

Reply:
Oh fancy. yeah I can do that. I didn't think much into making a sophisticated test here. I just wanted a non-zero way of showing my fixes the syntax of the URL. I can do this too though! Seems like a good idea.